content/docs/2.14/operate/cluster.md

```diff
@@ -129,6 +129,15 @@ The Kubernetes client config used within KEDA Operator and KEDA Metrics Adapter
 | kube-api-burst      | cfg.Burst               | 30            | Set the burst for throttling requests sent to the apiserver    |
 | disable-compression | cfg.DisableCompression  | true          | Disable compression for response in k8s restAPI in client-go, see [this Kubernetes issue](https://github.com/kubernetes/kubernetes/issues/112296) for details |
 
+## Metrics Service Parameters
+
+The Metrics Service config used by the KEDA Metrics Adapter to connect to the KEDA Operator can be adjusted by passing the following command-line flags to the Adapter binary:
```

Comment:
can we add a sentence please here right at the begining explainig what Metrics Service is? We should also  name this: "KEDA gRPC Metrics Service"